dataservice/util/data_gen/data_generator.py

```diff
@@ -310,3 +328,20 @@ def _create_diagnoses(self, total):
             }
             diag_list.append(Diagnosis(**data))
         return diag_list
+
+    def _create_phenotypes(self, total):
+        """
+        creates phenotypes
+        """
+        phen_list = []
+        for i in range(total):
+            ph = random.choice(self.phenotype_chosen_list)
+            phen = {
+                'external_id': 'phenotype_{}'.format(i),
+                'phenotype': ph[0],
```

Comment:
`ph[0]` will only get the first character of the string?

Reply:
i am selecting from the list of lists.  
![screen shot 2018-02-12 at 1 08 41 pm](https://user-images.githubusercontent.com/20447220/36111973-fc8262c0-0ff5-11e8-89f4-648b32a20054.png)